isochrones/dartmouth/tri.py

```diff
@@ -3,7 +3,7 @@
 import sys, os
 import pandas as pd
 import numpy as np
-import cPickle as pickle
+import _pickle as pickle
```

Comment:
The issue with this is that the code needs to work with both python 2 and 3.  Also, to my understanding in python 3 you should just import `pickle`, not `_pickle`.  So maybe something like this?
```
try:
    import cPickle as pickle
except ImportError:
    import pickle
```